eventfinder/src/eventcard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
+    textAlign: 'left',
+    transition: '0.3s',
+    '&:hover':{
+      transform: 'scale(1.02)',
+      transition: '0.3s'
+    }
+  },
+  media: {
+    height: 140,
+  },
+});
+
+export default function EventCard() {
+  const classes = useStyles();
+
+  return (
```

Comment:
While this definition works for this single card, it is not reusable. Repeating this structure for each event will be unsustainable, but thankfully React allows us to inject variables into the component.

Here is an example of how that might look:
```
function EventCard(props) {
  // The following is object destructuring, which is the same as accessing it via props.title, etc.
  const {
    image,
    title,
    date,
    description,
    etc
  } = props;

  return (
    <Card>
      ....
      <Typography>{title}</Typography>
      ...
    </Card>
  );
}
```

App.js
```
<div className="App">
  <header className="App-body">
    <Eventcard 
      title="Good News Everyone! ..."
      image="http://website.com/imagesrc.png"
      ...
    />
   </header>
</div>
```

Reply:
We can verify the accuracy of the props passed in by providing prop type validation checks that spit out errors to the console at runtime.

```
import PropTypes from 'prop-types';

...

function EventCard(props) {
...
}

EventCard.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string,
  numberOfAttendees: PropTypes.number.isRequired
};
```

Reply:
Changed format of function.